substratevm/src/native-image-maven-plugin/pom_template.xml

```diff
@@ -78,6 +78,16 @@
             <artifactId>svm-driver</artifactId>
             <version><svmVersion/></version>
         </dependency>
+        <dependency>
```

Comment:
You should not need this dependencies. They are provided transitively via `org.graalvm.nativeimage:svm-driver`.

Reply:
I had to. The POM of my `~.m2/repository` version of `org.graalvm.nativeimage:svm-driver` is empty:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <modelVersion>4.0.0</modelVersion>
  <groupId>org.graalvm.nativeimage</groupId>
  <artifactId>svm-driver</artifactId>
  <version>21.0.0-SNAPSHOT</version>
  <description>POM was created from install:install-file</description>
</project>
```

I ran `mx build`, `mx ideinit` and `mx maven-install`.

But if you are really sure it's not needed, I'll remove them.

Reply:
The trick is to run `mx maven-plugin-install --deploy-dependencies`. This will ensure you have all you need in your `~.m2/repository`.

Reply:
Thanks, that makes a difference. 

This subproject could clearly benefit from a README or CONTRIBUTING.md where you put the secret sauce 😉